concent-vm/install-golem-gui.yml

```diff
@@ -0,0 +1,35 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../concent-builder/repositories.yml
+    - vars.yml
+  tasks:
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:   Add node.js to repository
+          shell: curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
```

Comment:
Please don't execute scripts directly from the internet. There are better ways to install a package.

Reply:
This is a big and hairy script that boils down to adding a new repository to `sources.list`. I've moved it to `concent-builder/prerequire-configure-tasks.yml` and replaced with calls to `apt_repository`. No script needs to be sourced now.